buildo/github-workflow-pal
# #67: enforce topic on macro and sub issues (closes #67)

Issue #67

## Test Plan

- go to an issue page without topic, like #67
- you should see

![image](https://cloud.githubusercontent.com/assets/3280300/16158870/c4d6eeee-34bf-11e6-97dc-5ac8b50cbfb5.png)

- go to an issue page with topic (like #61), or add a topic (and reload the page)
- you should see the normal issue button

![image](https://cloud.githubusercontent.com/assets/3280300/16158915/08384eee-34c0-11e6-883f-5803da55ffb0.png)
